### ISSUE TITLE
[vitest/no-focused-tests] misses only.each and only.for

### ISSUE THREAD

**Issue by danielrentz** [vitest-dev/eslint-plugin-vitest#877]
The following code should trigger:

```js
// should trigger vitest/no-focused-tests
it.only.for([])(() => { })
it.only.each([])(() => { })
```

This works as expected:

```js
// already triggers vitest/no-disabled-tests
it.skip.for([])(() => { })
it.skip.each([])(() => { })
```
